pkg/standalone/run_test.go

```diff
@@ -281,4 +281,42 @@ func TestRun(t *testing.T) {
 		assert.Regexp(t, regexp.MustCompile(`( |^)--app-health-probe-timeout( |=)200`), argsFlattened)
 		assert.Regexp(t, regexp.MustCompile(`( |^)--app-health-threshold( |=)1`), argsFlattened)
 	})
+
+	t.Run("test setting defaults from struct tag", func(t *testing.T) {
+		basicConfig.AppPort = 0
```

Comment:
Are we testing the recursive scenario here, i.e. with struct within a struct?

Reply:
Yes, it will test the recursive scenario also with these fields - 
```
basicConfig.MaxConcurrency = 0
basicConfig.MaxRequestBodySize = 0
basicConfig.HTTPReadBufferSize = 0
```